### PR TITLE
Release version 0.5.12

### PR DESCRIPTION
**Pull request by benlower**
Bug fixes in npm package readme.

### CODE REVIEW

File: packages/ai-jsx/readme.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 [![Discord Follow](https://dcbadge.vercel.app/api/server/MsKAeKF8kU?style=flat)](https://discord.gg/MsKAeKF8kU)
 [![Twitter Follow](https://img.shields.io/twitter/follow/fixieai?style=social)](https://twitter.com/fixieai)
 
-[<img src="../docs/static/img/loom.png" alt="Image description" width="50%">](https://www.loom.com/share/c13b0c73f8d34f9f962048b39a4794f6?sid=a693f220-ccb1-4e70-913a-c2eda86dd0ce)
+## AI.JSX in Two Minutes
+
+https://github.com/fixie-ai/ai-jsx/assets/476553/301b79e4-7023-4adc-a3a5-72d5b7af0cde
```
Comment (NickHeiner) [inline packages/ai-jsx/readme.md:9]: image

This embeds very nicely in GH but I bet it'll just be a bare link on npm, which is the main place people will be reading this file.

File: packages/ai-jsx/readme.md [hunk 2]
```diff
@@ -27,7 +29,7 @@ For more details on how AI.JSX works with React in general, see our [AI+UI guide
 
 ## Examples
 
-You can play with live demos on our [live demo app](https://ai-jsx-nextjs-demo.vercel.app/) ([source](../nextjs-demo/)).
+You can play with live demos on our [live demo app](https://ai-jsx-nextjs-demo.vercel.app/) (source is available [here](./packages/nextjs-demo/)).
```
Comment (NickHeiner) [inline packages/ai-jsx/readme.md:32]: I for one really dislike the "link the word `here`" pattern. It's worse for screenreaders, among other things. But I won't hold up the PR on it. :smile:

Comment (benlower) [reply packages/ai-jsx/readme.md:32]: happy to change this. what would you suggest otherwise?

Comment (benlower) [reply packages/ai-jsx/readme.md:32]: will update this

File: packages/ai-jsx/readme.md (hunk 1 above)
Comment (benlower) [reply packages/ai-jsx/readme.md:9]: yes good point. will change this PR to stage in a test package first.

Comment (benlower) [reply packages/ai-jsx/readme.md:9]: investigating this...

**Pull request merged**